src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -28,6 +32,12 @@
   protected String sentinelPassword;
   protected String sentinelClientName;
 
+  protected boolean isRedisSslEnabled;
```

Comment:
`Master` would be better that `Redis`. E.g. `isMasterSslEnabled`. Also, apply this change entire PR.